pkg/oc/cli/cmd/set/buildhook.go

```diff
@@ -235,39 +219,44 @@ func (o *BuildHookOptions) Run() error {
 		return fmt.Errorf("%s/%s is not a build config", infos[0].Mapping.Resource, infos[0].Name)
 	}
 
-	if len(o.Output) > 0 || o.Local || kcmdutil.GetDryRunFlag(o.Cmd) {
-		return o.PrintObject(infos)
-	}
-
-	failed := false
+	allErrs := []error{}
 	for _, patch := range patches {
 		info := patch.Info
+		name := info.Name
+		if info.Mapping != nil {
+			name = fmt.Sprintf("%s/%s", info.Mapping.Resource.Resource, info.Name)
+		}
 		if patch.Err != nil {
-			fmt.Fprintf(o.Err, "error: %s/%s %v\n", info.Mapping.Resource, info.Name, patch.Err)
+			allErrs = append(allErrs, fmt.Errorf("error: %s %v\n", name, patch.Err))
 			continue
 		}
 
 		if string(patch.Patch) == "{}" || len(patch.Patch) == 0 {
-			fmt.Fprintf(o.Err, "info: %s %q was not changed\n", info.Mapping.Resource, info.Name)
+			fmt.Fprintf(o.ErrOut, "info: %s was not changed\n", name)
```

Comment:
I thought we were getting rid of this message?

Reply:
Turned into `glog.V(1).Infof("info: %s was not changed\n", name)`